lib/grplatt.gi

```diff
@@ -2349,7 +2349,7 @@ local uind,subs,incl,i,j,k,m,gens,t,c,p,conj,bas,basl,r;
     TryNextMethod();
   fi;
   uind:=IndexNC(G,U);
-  if uind<200 then
+  if uind<200 and ValueOption("usemaximals")<>true then
```

Comment:
This line is added by a commit with the commit message "Added Synonym: EmbeddedConjugates, EmbeddingConjugates". Is this change here intentional? If so, it probably should be in another commit...

Reply:
They are one commit as both minor additions to the intermediate subgroups code. I'll change the commit text.

Reply:
Thanks!